modules/monitoring-deploying-a-sample-service-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="deploying-a-sample-service-for-cluster-observability-operator_{context}"]
+= Deploying a sample service for Cluster Observability Operator
+
+This sample configuration deploys a sample service named `prometheus-coo-example-app` in the user-defined `ns1-coo` project. 
+The service exposes the custom `version` metric.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or the `monitoring-edit` cluster role.
+* The Cluster Observability Operator is installed.
+
+.Procedure
+
+. Create a service configuration YAML file named `prometheus-coo-example-app.yaml`.
+
+. Add the following deployment and service configuration details to the file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns1-coo
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: prometheus-coo-example-app
+  name: prometheus-coo-example-app
+  namespace: ns1-coo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prometheus-coo-example-app
+  template:
+    metadata:
+      labels:
+        app: prometheus-coo-example-app
+    spec:
+      containers:
+      - image: ghcr.io/rhobs/prometheus-coo-example-app:0.4.1
```

Comment:
```suggestion
      - image: ghcr.io/rhobs/prometheus-example-app:0.4.1
```
The image link doesn't change compared to the `Managing Metrics` section in the existing docs.

Reply:
Fixed